packages/cosmic-swingset/Makefile

```diff
@@ -109,15 +109,15 @@ scenario2-setup-nobuild:
 	$(AGCH) --home=t1/n0 init scenario2-chain --chain-id=$(CHAIN_ID)
 	# Init all the ag-solos, ensuring only the first one uses the $SOLO_MNEMONIC, if any.
 	set -e; \
-		first_solo=true; \
 		for port in `seq $(BASE_PORT) $$(($(BASE_PORT) + $(NUM_SOLOS) - 1))`; do \
 			NODE_OPTIONS= $(AG_SOLO) init t1/$$port --webport=$$port; \
-			if $$first_solo; then \
-				first_solo=false; \
-				sed -e "s!@FIRST_SOLO_ADDRESS@!$$(cat t1/$$port/ag-cosmos-helper-address)!g" \
-					'$(ECONOMY_TEMPLATE)' > '$(ECONOMY_PROPOSALS)'; \
+			if [ -z $$PRIMARY_ADDRESS ]; then \
+				echo "Getting PRIMARY_ADDRESS from agsolo config"; \
+				export PRIMARY_ADDRESS=$$(cat t1/$$port/ag-cosmos-helper-address); \
 			fi; \
-	  done
+	  done; \
+	  sed -e "s!@PRIMARY_ADDRESS@!$$PRIMARY_ADDRESS!g" '$(ECONOMY_TEMPLATE)' > '$(ECONOMY_PROPOSALS)'
+
```

Comment:
Please don't use whitespace-only lines in Makefiles.
```suggestion
```

Reply:
I see other line separators between targets so I assume you mean within a target. I don't see why but I've done it. Is the rationale documented? I don't see it in https://github.com/Agoric/agoric-sdk/wiki/Coding-Style

Reply:
Superstition. ;)